packages/sw-broadcast-cache-update/src/lib/behavior.js

```diff
@@ -19,30 +19,31 @@ import responsesAreSame from './responses-are-same';
 import {defaultHeadersToCheck, defaultSource} from './constants';
 
 /**
- * @memberof module:sw-broadcast-cache-update
+ * Can be used to compare two [Responses](https://developer.mozilla.org/en-US/docs/Web/API/Response)
+ * and uses the {@link https://developers.google.com/web/updates/2016/09/broadcastchannel|Broadcast Channel API}
+ * to notify interested parties when those Responses differ.
+ *
+ * For efficiency's sake, the underlying response bodies are not compared;
+ * only specific response headers are checked.
  *
- * @example
- * // Used as an automatically invoked as "behavior" by a RequestWrapper:
+ * @example <caption>Added as a "behavior" to a RequestWrapper to
+ * automatically dispatch messages</caption>
```

Comment:
What are you referring to here? If it's caption tag its there as a styled description of the code snippet.